### ISSUE TITLE
Stellar Expert doesn't find existing testnet account

### ISSUE THREAD

**Issue by ire-and-curses** [orbitlens/stellar-expert-explorer#154]
Account: `GB34PTUQQPYKN7XPWFZCIP77X5ZAEQEB3IN227EHIZFGLUUDS2Y2JVRW`

Stellar expert lookup is empty: https://stellar.expert/explorer/testnet/account/GB34PTUQQPYKN7XPWFZCIP77X5ZAEQEB3IN227EHIZFGLUUDS2Y2JVRW

Horizon info is populated: https://horizon-testnet.stellar.org/accounts/GB34PTUQQPYKN7XPWFZCIP77X5ZAEQEB3IN227EHIZFGLUUDS2Y2JVRW

Output of `lumen info`:
```
Ψ:~$ lumen set config:network test
Ψ:~$ lumen info GB34PTUQQPYKN7XPWFZCIP77X5ZAEQEB3IN227EHIZFGLUUDS2Y2JVRW
{
  "address": "GB34PTUQQPYKN7XPWFZCIP77X5ZAEQEB3IN227EHIZFGLUUDS2Y2JVRW",
  "balances": [
    {
      "asset": {
        "code": "SMX",
        "issuer": "GBXK4HMOMUA27BRDXM5AKVQI2VUZTN66FJOMMLTIHXBP5VUPCWXDSW7R",
        "type": "credit_alphanum4"
      },
      "amount": "100000.0000000",
      "limit": "922337203685.4775807"
    },
    {
      "asset": {
        "code": "USD",
        "issuer": "GCKFBEIYV2U22IO2BJ4KVJOIP7XPWQGQFKKWXR6DOSJBV7STMAQSMTGG",
        "type": "credit_alphanum4"
      },
      "amount": "14523.0000000",
      "limit": "922337203685.4775807"
    },
    {
      "asset": {
        "code": "ARS",
        "issuer": "GB7TAYRUZGE6TVT7NHP5SMIZRNQA6PLM423EYISAOAP3MKYIQMVYP2JO",
        "type": "credit_alphanum4"
      },
      "amount": "0.0000000",
      "limit": "922337203685.4775807"
    }
  ],
  "signers": [
    {
      "public_key": "",
      "weight": 1,
      "key": "GB34PTUQQPYKN7XPWFZCIP77X5ZAEQEB3IN227EHIZFGLUUDS2Y2JVRW",
      "type": "ed25519_public_key"
    }
  ],
  "flags": {
    "auth_required": false,
    "auth_revocable": false,
    "auth_immutable": false
  },
  "native_balance": {
    "asset": {
      "code": "XLM",
      "issuer": "",
      "type": "native"
    },
    "amount": "18057.4318725",
    "limit": ""
  },
  "home_domain": "",
  "thresholds": {
    "high": 0,
    "medium": 0,
    "low": 0
  },
  "data": {},
  "seq": "5110036124663813"
}
```

**Comment by ire-and-curses** [comment]
This bug seems potentially very urgent to me.

**Comment by orbitlens** [comment]
Caused by new `PATH_PAYMENT_STRICT_SEND` op XDR on the testnet.
Fixed in v0.11.2.
